scripts/ingest_regress_test_result-new-format.py

```diff
@@ -134,7 +134,7 @@ def ingest_test_result(
             if p["name"].startswith("__")
         }
         arch = parameters.get("arch", "UNKNOWN").strip("'")
-        lfc = parameters.get("lfc", "False") == "True"
+        lfc = parameters.get("lfc", "without-lfc") == "with-lfc"
```

Comment:
We got only `false` in the DB for the last commit:
```
SELECT DISTINCT lfc FROM results WHERE revision = 'e49484f4aefcaaae3afdab1d1f5168853ccf9fa4';
 lfc
-----
 f
(1 row)
```

Probably we need to strip additional quotes:

```suggestion
        lfc = parameters.get("lfc", "without-lfc").strip("'") == "with-lfc"
```

Reply:
It works!

```
SELECT DISTINCT lfc FROM results WHERE revision = '7b465c233e77d048c35a372fc8484367a8981c37';
 lfc
-----
 f
 t
(2 rows)
```